server/src/models/characterModel.ts

```diff
@@ -270,6 +350,21 @@ characterSchema.pre('save', function (next) {
   next()
 })
 
-const Character = mongoose.model('Character', characterSchema)
+characterSchema.pre('save', async function (next) {
+  const user = await User.findById(this.createdBy)
```

Comment:
try catch block in case the codex goes wrong?

Reply:
if you see anymore try catch blocks that need to be there let me know, i never really know when its good to use. or should i just always put one in?